onnx_mxnet/import_onnx.py

```diff
@@ -206,6 +212,20 @@ def _fix_slice(self, inputs, new_attr):
                 slice_op = mx.sym.slice_axis(slice_op, axis=axis, begin=begin[i], end=end[i])
         return slice_op
 
+    def _fix_squeeze(self, inputs, new_attr):
```

Comment:
Should we create an issue in MXNet to add support for squeeze operator and add a link to the issue here in comment?

Reply:
MXNet added it last week , but is not part of current release plan. Once it gets in release we can change this functionality.